checks/smtp/smtp_test.go

```diff
@@ -0,0 +1,23 @@
+package smtp
+
+import (
+	"context"
+	"net/smtp"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestStatusUp(t *testing.T) {
+	// Arrange
+	client, err := smtp.Dial("localhost:25")
+	require.NoError(t, err)
+
+	check := TestConnection(client)
```

Comment:
Change to `New` here as well?
```suggestion
	check := New(client)
```